### PR TITLE
Add `nimsite.uk`

### PR DESCRIPTION
**Pull request by kezdev**
Public Suffix List (PSL) Pull Request (PR) Template
====

Each PSL PR needs to have a description, rationale, indication of DNS validation and syntax checking, as well as a number of acknowledgements from the submitter.  This template must be included with each PR, and the submitting party MUST provide responses to all of the elements in order to be considered.

### Checklist of required steps

* [X] Description of Organization
* [X] Robust Reason for PSL Inclusion
* [X] DNS verification via dig
* [X] Run Syntax Checker (make test)

* [X] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration, and we shall keep the \_PSL txt record in place in the respective zone(s) in the affected section

__Submitter affirms the following:__ 

  * [X] We are listing *any* third-party limits that we seek to work around in our rationale such as those between IOS 14.5+ and Facebook (see [Issue #1245](https://github.com/publicsuffix/list/issues/1245) as a well-documented example)

  * [X] This request was _not_ submitted with the objective of working around other third-party limits

  * [X] The [Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines) were carefully _read_ and _understood_, and this request conforms
  * [X] The submission follows the [guidelines](https://github.com/publicsuffix/list/wiki/Format) on formatting and sorting

---

For Private section requests that are submitting entries for domains that match their organization website's primary domain, please understand that this can have impacts that may not match the desired outcome and take a long time to rollback, if at all.

To ensure that requested changes are entirely intentional, make sure that you read the affectation and propagation expectations, that you understand them, and confirm this understanding. 

PR Rollbacks have lower priority, and the volunteers are unable to control when or if browsers or other parties using the PSL will refresh or update.

(Link: [about propagation/expectations](https://github.com/publicsuffix/list/wiki/Guidelines#appropriate-expectations-on-derivative-propagation-use-or-inclusion))

 * [X] *Yes, I understand*.  I could break my organization's website cookies etc. and the rollback timing, etc is acceptable.  *Proceed*.
---

Description of Organization
====
I am Kez Hall, a Web Developer at Nimbus Hosting, a UK-based web hosting company that prioritises carbon neutrality. Our key feature is a unique platform that enables users to manage all their clients' websites from one centralised platform.

Organization Website: [https://nimbushosting.co.uk](https://nimbushosting.co.uk)

Reason for PSL Inclusion
====
Our customers are given a unique subdomain of `nimsite.uk` for each website they create on the platform. This act's as a instant/demo/development URL. The platform uses cookies & we need to be able to provide isolation between each users environments.

We have previously added to the list in 2017: https://github.com/publicsuffix/list/pull/491

Number of users this request is being made to serve: thousands

DNS Verification via dig
=======
```
$ dig +short TXT _psl.nimsite.uk
https://github.com/publicsuffix/list/pull/1797
```

### CODE REVIEW

**Comment by dnsguru** [comment]
@nickdford please comment as original submitter of #491

**Comment by nickdford** [comment]
Commenting as per your request @dnsguru

**Comment by dnsguru** [comment]
> Commenting as per your request @dnsguru

Apologize for not being more clear about the ask....  Was that a "This is OK. LGTM" or a "This is not OK" comment, as that's what the ask was.

**Comment by nickdford** [comment]
Sorry, @dnsguru . all good here!
I'm happy with @kezdev 's request to be merged.

**Comment by kezdev** [comment]
@dnsguru is this ok to be merged now?

File: public_suffix_list.dat [hunk 1]
```diff
@@ -12591,6 +12591,7 @@ ngrok.pizza
 // Nimbus Hosting Ltd. : https://www.nimbushosting.co.uk/
 // Submitted by Nicholas Ford <nick@nimbushosting.co.uk>
```
Comment (simon-friedberger) [inline public_suffix_list.dat:12592]: Please use a non-personal e-mail address.

**Review by simon-friedberger — CHANGES_REQUESTED** [review]
* [x] Expiration (Note: Must STAY >2y at all times)
  * [x] nimsite.uk expires 2027-01-19
  * [x] nh-serv.co.uk expires 2027-06-02
* [x] DNS _psl entries
* [x] Tests pass
* [x] Sorting
* [x] Reasoning/Organization description

File: public_suffix_list.dat (hunk 1 above)
Comment (nickdford) [reply public_suffix_list.dat:12592]: @simon-friedberger this is a work email address.
Would you rather an alias or shared address?

Comment (simon-friedberger) [reply public_suffix_list.dat:12592]: Shared address; something that would stay valid if you left the company.

Comment (kezdev) [reply public_suffix_list.dat]: @nickdford @simon-friedberger I've updated the email to a shared address. Please review

**Comment by kezdev** [comment]
> * [ ]  Expiration (Note: Must STAY >2y at all times)
>   
>   * [ ]  nimsite.uk expires 2025-01-19
>   * [ ]  nh-serv.co.uk expires 2025-06-02
> * [x]  DNS _psl entries
> * [x]  Tests pass
> * [x]  Sorting
> * [x]  Reasoning/Organization description

@simon-friedberger We've renewed both domains for another 2 years. The new expiry dates should be:

nimsite.uk expires 2027-01-18
nh-serv.co.uk expires 2027-06-01

**Comment by kezdev** [comment]
@simon-friedberger

**Review by simon-friedberger — APPROVED** [review]

**Comment by kezdev** [comment]
Thanks @simon-friedberger 🙏🏻 

Do you have an ETA for when this will be rolled out?

**Comment by simon-friedberger** [comment]
Once it is merged, it will be used in the next build of Firefox. A build usually takes about 2 months to go from nightly to beta to release. I heard Chrome updates more infrequently. For other browsers I have no information.

**Comment by xnetco** [comment]
@simon-friedberger Do you need anything from us to merge this?

**Pull request merged**